ClientLib/src/main/java/org/droidplanner/services/android/impl/core/drone/autopilot/apm/solo/ArduSolo.java

```diff
@@ -172,6 +173,26 @@ public SoloComp getSoloComp() {
         return soloComp;
     }
 
+    @Override
+    public void onMavLinkMessageReceived(MAVLinkMessage message) {
```

Comment:
I would be expecting RADIO_STATUS should be handled in GenericMavLinkDrone (not ArduPilot) as  its a common MAVLink message https://github.com/dronekit/dronekit-android/blob/develop/ClientLib/src/main/java/org/droidplanner/services/android/impl/core/drone/autopilot/generic/GenericMavLinkDrone.java#L601

Either solution would work, but I think when I added the exception method the idea was that is should be used to override extra messages that you need to let through. (You may need to use processRadio... method to normalize the values)

The code for filter messages was adding as part of the plan to allow sys_id other than 1 to work.. That's the next step at least.